app/controllers/api/messaging.php

```diff
@@ -3613,38 +3679,6 @@
             $message->setAttribute('status', $status);
         }
 
-        if (!\is_null($scheduledAt)) {
-            if (\is_null($message->getAttribute(('scheduleId')))) {
-                $schedule = $dbForConsole->createDocument('schedules', new Document([
-                    'region' => App::getEnv('_APP_REGION', 'default'),
-                    'resourceType' => 'message',
-                    'resourceId' => $message->getId(),
-                    'resourceInternalId' => $message->getInternalId(),
-                    'resourceUpdatedAt' => DateTime::now(),
-                    'projectId' => $project->getId(),
-                    'schedule'  => $scheduledAt,
-                    'active' => $status === MessageStatus::SCHEDULED,
-                ]));
-
-                $message->setAttribute('scheduleId', $schedule->getId());
-            } else {
-                $schedule = $dbForConsole->getDocument('schedules', $message->getAttribute('scheduleId'));
-
-                if ($schedule->isEmpty()) {
-                    throw new Exception(Exception::SCHEDULE_NOT_FOUND);
-                }
-
-                $schedule
-                    ->setAttribute('resourceUpdatedAt', DateTime::now())
-                    ->setAttribute('schedule', $scheduledAt)
-                    ->setAttribute('active', $status === MessageStatus::SCHEDULED);
-
-                $dbForConsole->updateDocument('schedules', $schedule->getId(), $schedule);
-            }
-
-            $message->setAttribute('scheduleId', $schedule->getId());
-        }
-
```

Comment:
Can we add a test for canceling the schedule?

Reply:
Will do :+1: